templates/boilerplate-template/README.md

```diff
@@ -22,6 +22,7 @@ To get started with the Boilerplate template and learn more about the template f
 - Aptos TS SDK
 - Aptos Wallet Adapter
 - Node based Move commands
+- [Vite-pwa](https://vite-pwa-org.netlify.app/)
```

Comment:
could we add a section explaining the end user 
1. what is PWA briefly
2. once they have the boilerplate generated locally, what should they do?
3. how to deploy it?

Reply:
updated doc on learn

Reply:
updated doc on learn

Reply:
updated doc on learn

Reply:
updated doc on learn